wagtail/wagtailadmin/static_src/wagtailadmin/scss/core.scss

```diff
@@ -417,19 +417,76 @@ footer,
             display: none;
         }
 
-        img {
-            width: 60px;
+        .wagtail-logo-container {
             float: none;
             margin: auto;
             display: block;
```

Comment:
I don't think those three (`float`, `margin`, `display`) attributes are necessary anymore, since this element is now a `div` not an `img`.